src/app/gateway/gateway-table/gateway-table.component.html

```diff
@@ -50,6 +50,69 @@
       </td>
     </ng-container>
 
```

Comment:
If i use a smaller screen (such as laptop), then the table will countinue out of the box. It looks pretty bad..
![image](https://github.com/OS2iot/OS2IoT-frontend/assets/70511721/c5d4f86e-57b9-443c-80bd-121adce0942c)